src/main/resources/frontend/src/services/configService.ts

```diff
@@ -37,7 +28,7 @@ export const setLivePhases = async (phases: Array<Phase>): Promise<void> => {
 }
 
 export const setCanvasCourseIds = async (): Promise<void> => {
-    await doSetConfigItem("GET", "/api/admin/config/courseIds", null);
+  await doSetConfigItem("GET", "/api/admin/config/courseIds", {});
```

Comment:
It looks like we have some whitespace inconsistency going on... Some of the lines in this file are intended with 2 spaces, but others are indented with 4. Just pick a value and stick with it.

Just we look into setting up a linter that will automatically correct these kinds of details whenever the code is saved?

Reply:
For the most part I've been matching whatever is already in the file without thinking about it. If we had a linter that would be great, especially since we don't actually have a style guide